data-loader-service/src/main/kotlin/cz/kss/proj/dataloaderservice/config/CustomerConfig.kt

```diff
@@ -0,0 +1,15 @@
+package cz.kss.proj.dataloaderservice.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.reactive.function.client.WebClient
+
+@Configuration
+class CustomerConfig {
+    @Bean
+    fun customerWebClient(webClientBuilder: WebClient.Builder): WebClient {
+        return webClientBuilder
+            .baseUrl("http://localhost:8881")
```

Comment:
The property should be read from a config file. Something like this:

```
@Value("\"server.base.url\""
private lateinit var baseUrl: String
```

Reply:
I fixed it